app/frontend/src/components/forms/SubmissionsTable.vue

```diff
@@ -41,7 +41,6 @@ export default {
           key: 'event',
         },
       ],
-      forceTableRefresh: 0,
```

Comment:
Could this be returned to it's original position? The variable names should be sorted in alphabetical order for consistency.

Reply:
Okay